src/test/run-make/track-path-dep-info/macro_def.rs

```diff
@@ -0,0 +1,11 @@
+#![feature(track_path)]
+#![crate_type = "proc-macro"]
+
+extern crate proc_macro;
+use proc_macro::*;
+
+#[proc_macro]
+pub fn access_tracked_paths(_: TokenStream) -> TokenStream {
+    let _ = tracked_path::path("emojis.txt");
```

Comment:
```suggestion
    tracked_path::path("emojis.txt");
```
(This function doesn't return anything.)